compute_tools/src/monitor.rs

```diff
@@ -165,6 +165,31 @@ fn watch_compute_activity(compute: &ComputeNode) {
                         continue;
                     }
                 }
+                //
+                // Don't suspend compute if there is activity in physical replication
+                //
+                let physical_replication_query =
+                    "select last_msg_receipt_time from pg_stat_wal_receiver;";
```

Comment:
> We currently don't check if there's any physical replication happening, which can cause the compute to get suspended during a RO replica benchmark!

Does it mean that with this change we won't suspend RO if RW is running and got some writes?

If so, I'm not sure that the desired behavior, as we now create ephemeral ROs and this change means they may never suspend if RW is active

Reply:
Yes you and Stas are right, this is probably not the right fix